test/framework/cluster.go

```diff
@@ -1046,7 +1046,7 @@ func (e *ClusterE2ETest) managementCluster() *types.Cluster {
 
 // KubeconfigFilePath retrieves the Kubeconfig path used for the workload cluster.
 func (e *ClusterE2ETest) KubeconfigFilePath() string {
-	return filepath.Join(e.ClusterName, fmt.Sprintf("%s-eks-a-cluster.kubeconfig", e.ClusterName))
+	return filepath.Join(e.ClusterConfigFolder, fmt.Sprintf("%s-eks-a-cluster.kubeconfig", e.ClusterName))
```

Comment:
In most cases the folder is the same as the cluster name, but it doesn't have to be
This removes the assumption and writes all files to the same folder consistently